src/resources/subscriptions.js

```diff
@@ -34,7 +36,123 @@ const fetch = async (publicKey, withAccountData = false, transactionId = undefin
   return NinaClient.get(`/subscriptions/${publicKey}`, transactionId ? { transactionId } : undefined);
 };
 
+/**
+ * @function subscriptionSubscribe
+ * @param {Object} client the NinaClient instance
+ * @param {String} subscribeToAccount the account to subscribe to
+ * @param {String} hubHandle the hub handle to subscribe to
+ * @returns {Object} the Subscription data
+ */
+
+const subscriptionSubscribe = async (client, subscribeToAccount, hubHandle) => {
+  try {
+  const {provider} = client
+  const program = await client.useProgram()
+  subscribeToAccount = new anchor.web3.PublicKey(subscribeToAccount)
+
+const [subscription] = await anchor.web3.PublicKey.findProgramAddress(
+  [
+    Buffer.from(anchor.utils.bytes.utf8.encode('nina-subscription')),
+    provider.wallet.publicKey.toBuffer(),
+    subscribeToAccount.toBuffer(),
+  ],
+  program.programId
+)
+
+let tx;
+if (hubHandle) {
+  tx = await program.methods
+    .subscriptionSubscribeHub(hubHandle)
+    .accounts({
+      payer: provider.wallet.publicKey,
+      from: provider.wallet.publicKey,
+      subscription,
+      to: subscribeToAccount,
+      systemProgram: anchor.web3.SystemProgram.programId,
+    })
+    .transaction()
+} else {
+  tx = await program.methods.subscriptionSubscribeAccount()
+  .accounts({
+    payer: provider.wallet.publicKey,
+    from: provider.wallet.publicKey,
+    subscription,
+    to: subscribeToAccount,
+    systemProgram: anchor.web3.SystemProgram.programId,
+  })
+  .transaction()
+}
+
+tx.recentBlockhash = (await provider.connection.getRecentBlockhash()).blockhash
+
+tx.feePayer = provider.wallet.publicKey
+
+const txid = await provider.wallet.sendTransaction(tx, provider.connection)
+await getConfirmTransaction(txid, provider.connection)
+const subscriptionData = await fetch(subscription.toBase58(), txid)
+return {
+  subscriptionData: subscriptionData,
```

Comment:
return prop should be called name of resource it is returning - so here `subscription`